test/unit/Reflection/ReflectionClassTest.php

```diff
@@ -206,6 +206,44 @@ public function testGetProperties() : void
         self::assertCount(4, $properties);
     }
 
+    public function testGetPropertiesReturnsInheritedProperties() : void
+    {
+        $reflector = new ClassReflector(new SingleFileSourceLocator(__DIR__ . '/../Fixture/InheritedClassProperties.php'));
+        $classInfo = $reflector->reflect('Qux');
```

Comment:
I'd love to have in CI.

Could you share the name? I'm unable to guess it: https://github.com/kalessil/phpinspectionsea/tree/master/src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors